### PR TITLE
fix: formatting textbox behavior symbols

### PR DESCRIPTION
**Pull request by Ouillyang**
GitHub Issue: #

## Proposed Changes

 - Bug fix

## What is the current behavior?

The behavior does not show up in the options when we're calling the correct namespace

## What is the new behavior?

It should have the appropriate symbol for the behavior to appear

## Checklist

Please check if your PR fulfills the following requirements:

- [ ] Documentation has been added/updated
- [ ] Automated Unit / Integration tests for the changes have been added/updated
- [ ] Contains **NO** breaking changes
- [ ] Updated the Changelog
- [ ] Associated with an issue

### CODE REVIEW

File: src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿#if NETFX_CORE || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS_UWP
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ || WINDOWS || WINDOWS_WINUI || WINUI
```
Comment (jeanplevesque) [inline src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: Are there platforms where the behavior is not supported? Couldn't we just remove this `#if`?

Comment (juliecantin) [inline src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: Does this lead to the file being included in any unwanted platform? Which platforms does it currently build on?

**Review by juliecantin — APPROVED** [review]

File: src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs (hunk 1 above)
Comment (Ouillyang) [reply src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: Is there a way to check for all platforms? I mostly copied the symbols from `TextBoxBehavior` and tried to build solution.

Comment (Ouillyang) [reply src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: Is there a way to check for all platforms? I mostly copied the symbols from `TextBoxBehavior` and tried to build solution.

Comment (juliecantin) [reply src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: look at the csproj https://github.com/nventive/Nventive.View/blob/master/src/View.Uno.WinUI/View.Uno.WinUI.csproj

Comment (Ouillyang) [reply src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: I switched through every platform and there was no errors/issues, I'm thinking of removing the `#if` like JP said.

Comment (Ouillyang) [reply src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: It looks like it's supported on all platforms, I've removed the `#if`

Comment (Ouillyang) [reply src/View.Uno.Shared/Extensions/TextBox/FormattingTextBoxBehavior.cs]: Tested without the `#if` and it works on all the platforms.
Edit: never mind it looks like it's crashing for iOS

**Review by jeanplevesque — APPROVED** [review]

**Review by Soap-141 — APPROVED** [review]

**Comment by Ouillyang** [comment]
Looks like it's crashing on startup on iOS
image

More specifically this line
image

**Comment by Soap-141** [comment]
> Looks like it's crashing on startup on iOS image
> 
> More specifically this line image

It could be the namespace, Microsoft.Xaml.Control / Windows.Xaml etc... (WinUI vs UWP)

**Comment by Ouillyang** [comment]
We do have this at the top of the file
image

**Comment by Soap-141** [comment]
> We do have this at the top of the file image

Check what is "d"

**Comment by Ouillyang** [comment]
> > We do have this at the top of the file image
> 
> Check what is "d"

image

image
Really weird, since it's casted to TextBox right above it

**Comment by Soap-141** [comment]
> > > We do have this at the top of the file image
> > 
> > 
> > Check what is "d"
> 
> image
> 
> image Really weird, since it's casted to TextBox right above it

You should add a breakpoint and check what is the value because your cast is failing so it's definitely a bad value null or another type.

**Comment by Ouillyang** [comment]
I'm having trouble with the debugger unfortunately.. the app crashes even when it reaches the breakpoint.
image

**Comment by Ouillyang** [comment]
Looks like it was an issue with my VS, somehow building with old files/changes... I restarted and it runs on iOS just fine...
Thanks for your help though!

**Pull request merged**